ui/rdb/views/CommandsView.ts

```diff
@@ -0,0 +1,45 @@
+class CommandsView {
+    public convertArgs(inputArgs, requestArgsSchema) {
+        const args = [];
+        for (const key of inputArgs.propertyOrder) {
+            let rawArg = inputArgs.properties[key];
+            // Handles references to other definitions in schema
+            if (rawArg['$ref']) {
+                const refParts = rawArg['$ref'].split("/");
+                let rawArgRef = requestArgsSchema;
+
+                for (const key of refParts) {
+                    // Skip first entry
+                    if (key === '#') continue;
+                    rawArgRef = rawArgRef[key];
+                }
+                for (const key in rawArgRef) {
+                    rawArg[key] = rawArgRef[key];
+                }
+            }
+            args.push({
+                ...rawArg,
+                id: key
+            });
+        }
+        return args;
+    }
+
+    public processArgToSendToExtension(argType: string, argValue: string) {
+        if (argType == 'boolean') {
+            return argValue === 'true'
```

Comment:
What about `'false'`? Shouldn't you parse the argValue to see which it should be in this case?

Reply:
'false' does not equal 'true' so would convert to `false` as would anything else that does not equal 'true'. Does that make sense?